ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -857,3 +860,23 @@ Verify Model KServe Deployment
     ${n_containers}=    Get Length    ${containers}
     Run Keyword And Continue On Failure    Verify Deployment    ${model_pods}  ${exp_replicas}
     ...    ${n_containers}  ${containers}
+
+Wait For ISVC To Be Loaded
```

Comment:
I think that if you combine the keyword `Wait Until Keyword Succeeds` with a new keyword `ISVC Should Be Loaded` the code could be much simpler

Example:

```
Wait For ISVC To Be Loaded
    [Documentation]    Waits for the ISVC to be Loaded (i.e., Success status in the grid)
    [Arguments]    ${isvc_name}    ${namespace}    ${retries}=36
    Wait Until Keyword Succeeds   retry=${retries}    5s    ISVC Should Be Loaded    ${isvc_name}    ${namespace}
```

Reply:
Hehe, I knew there was something like this but I completely forgot the name of this, I was searching for it already for this PR to comment, but failed: #1518.

Reply:
Done in 0030dd0
Thanks @jgarciao and @jstourac  for the recomendations :)